esx_service/cli/vmdkops_admin.py

```diff
@@ -255,6 +255,20 @@ def commands():
         'status': {
             'func': status,
             'help': 'Show the status of the vmdk_ops service'
+        },
+        'edit': {
```

Comment:
'edit' sounds like we are editing content. 'settings' or 'config' sounds better to me . `config --options`